src/smc.cpp

```diff
@@ -144,21 +144,33 @@ double add_constraint(const std::string& name, List constraints,
  * Add specific constraint weights & return the cumulative weight vector
  */
 vec get_wgts(const umat &districts, int n_distr, int distr_ctr, bool final,
-             double alpha, vec &lp, const uvec &pop, List constraints,
-             const uvec &counties, int n_cty, int verbosity) {
+             double alpha, vec &lp, const uvec &pop, double parity, const Graph g,
+             List constraints, int verbosity) {
     int V = districts.n_rows;
     int N = districts.n_cols;
 
     for (int i = 0; i < N; i++) {
         if (constraints.size() == 0) continue;
 
+        lp[i] += add_constraint("population", constraints,
```

Comment:
any way to rename this one, since we're changing the interface? `pop_dev` or something would be more descriptive (applies to flip and M-S too, obviously)

Reply:
Yeah, let's do pop_dev.